Web3/Classes/ContractABI/Contract/ContractPromiseExtensions.swift

```diff
@@ -18,19 +18,19 @@ import PromiseKit
 
 public extension SolidityInvocation {
 
-    public func call(block: EthereumQuantityTag = .latest) -> Promise<[String: Any]> {
+    func call(block: EthereumQuantityTag = .latest) -> Promise<[String: Any]> {
```

Comment:
Ok is this a new Swift 4.2 thing? Are functions inside public extensions automatically public?

Reply:
yes. As I understood this is Swift 4 thing but in Swift 4.2 they added warnings for "public in public" declaration

Reply:
Got it now. Thanks.